### PR TITLE
chore(wrw): config changes addition of avaxc coin

### PR DESCRIPTION
**Pull request by nrjsuthar**
COIN-1708

### CODE REVIEW

File: src/helpers/config.ts [hunk 1]
```diff
@@ -819,65 +821,89 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
...
       allCoinMetas.tarbeth,
       allCoinMetas.topeth,
       allCoinMetas.tpolygon,
+      allCoinMetas.tavaxc,
     ] as const,
     arbeth: [
       allCoinMetas.eth,
       allCoinMetas.polygon,
       allCoinMetas.opeth,
+      allCoinMetas.avaxc,
```
Comment (kamleshmugdiya) [inline src/helpers/config.ts:888]: avaxc should not be in intended chain for any of the wrong chain.

File: src/helpers/config.ts [hunk 2]
```diff
@@ -819,65 +821,89 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
...
       allCoinMetas.opeth,
       allCoinMetas.polygon,
+      allCoinMetas.avaxc,
     ] as const,
     tbaseeth: [
       allCoinMetas.hteth,
       allCoinMetas.tarbeth,
       allCoinMetas.topeth,
       allCoinMetas.tpolygon,
+      allCoinMetas.tavaxc,
```
Comment (kamleshmugdiya) [inline src/helpers/config.ts:882]: avaxc should not be in intended chain for any of the wrong chain.

File: src/helpers/config.ts [hunk 3]
```diff
@@ -819,65 +821,89 @@ export const evmCCRIntendedChainCoins: Record<string, readonly CoinMetadata[]> =
...
     ] as const,
     baseeth: [
       allCoinMetas.eth,
       allCoinMetas.arbeth,
       allCoinMetas.opeth,
       allCoinMetas.polygon,
+      allCoinMetas.avaxc,
```
Comment (kamleshmugdiya) [inline src/helpers/config.ts:875]: same

**Review by kamleshmugdiya — CHANGES_REQUESTED** [review]

File: src/helpers/config.ts [hunk 4]
```diff
@@ -928,6 +954,7 @@ export const tokenParentCoins = {
   topethToken: 'topeth',
   polygonToken: 'polygon',
   tpolygonToken: 'tpolygon',
+  tavaxchainToken: 'tavaxchain'
```
Comment (venkateshv1266) [inline src/helpers/config.ts:957]: Why this is required?

Comment (nrjsuthar) [reply src/helpers/config.ts:957]: This is not required, was checking for options on WRW. Removed.

**Review by kamleshmugdiya — DISMISSED** [review]

**Review by kamleshmugdiya — APPROVED** [review]

**Comment by venkateshv1266** [comment]
Can u please squash all 4 commits?

**Comment by nrjsuthar** [comment]
Working on the same @venkateshv1266

**Review by kamleshmugdiya — CHANGES_REQUESTED** [review]
+1 please squash your commits.

**Comment by nrjsuthar** [comment]
@kamleshmugdiya @venkateshv1266 Done

**Review by kamleshmugdiya — DISMISSED** [review]

**Comment by venkateshv1266** [comment]
Please squash the commits

**Comment by venkateshv1266** [comment]
Remove `yarn.lock` file. Since we use npm to install dependencies in wrw, `yarn.lock` file is not required.

**Pull request merged**